web-registry/src/pages/Projects/hooks/useProjectsSellOrders.tsx

```diff
@@ -52,42 +55,46 @@ const getProjectDisplayData = async (
   limit: number,
 ): Promise<ProjectWithOrderData[]> => {
   const projectsWithOrderData = await Promise.all(
-    projects.slice(0, limit).map(async (project: ProjectInfo) => {
-      const sellOrdersNormalized = sellOrders
-        .filter(sellOrder => sellOrder?.batchDenom?.startsWith(project.id))
-        .map(sellOrder => {
-          return {
-            ...sellOrder,
-            id: String(sellOrder.id),
-          };
-        });
-      const purchaseInfo = getPurchaseInfo(project.id, sellOrders);
-      let metadata;
-      if (project.metadata.length) {
-        try {
-          metadata = await getMetadata(project.metadata);
-        } catch (error) {
-          // eslint-disable-next-line
-          console.error(error);
+    projects
+      .sort(sortProjectsBySellOrdersAvailability(sellOrders))
```

Comment:
@mhagel I double check this and it is still needed. If you want to check you can try giving a limit lower to the number of projects with sell orders + metadata.

Reply:
sounds good